### DISCUSSION TITLE
Where will I be able to buy XCH?

### DISCUSSION THREAD

**Discussion by GeorgePapageorgakis** [Chia-Network/chia-blockchain#1856] [General]
So the coin is to become available at the end of April if I remember correctly. Do we have any updates on which DEX or CEX it will become available?

**Comment by hoffmang9** [comment]
All we can say about which exchanges and when is on page 23 - https://www.chia.net/assets/Chia-Business-Whitepaper-2021-02-09-v1.0.pdf

**Reply by third4ut3** [reply]
How can buy this coin?

**Reply by wanghongqi8548488** [reply]
还不可以加以 5月份吧

**Reply by GeorgePapageorgakis** [reply]
> 
> 
> All we can say about which exchanges and when is on page 23 - https://www.chia.net/assets/Chia-Business-Whitepaper-2021-02-09-v1.0.pdf

No date? Nothing? Stealth sales?

**Reply by dorkmo** [reply]
once transactions are enabled, you could probably sell face to face transactions. any DEX or regular exchange will have to do a bit or work to implement the chia wallet stuff, so its really up to them on a date. i believe chia corporate is trying to reach out to some bigger exchanges to have something ready, but i wouldnt expect a date until its closer to being ready.

**Reply by dorkmo** [reply]
i'd like to see bisq as an option but i dont think theyre adding any alt coins at the moment

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
